core/server/proxy/src/main/java/alluxio/proxy/s3/S3RestServiceHandler.java

```diff
@@ -891,11 +891,7 @@ public Response createObjectOrUploadPart(@HeaderParam("Authorization") final Str
             String entityTag = Hex.encodeHexString(digest);
             // persist the ETag via xAttr
             // TODO(czhu): compute the ETag prior to creating the file to reduce total RPC RTT
-            fs.setAttribute(objectUri, SetAttributePOptions.newBuilder()
-                .putXattr(S3Constants.ETAG_XATTR_KEY,
-                    ByteString.copyFrom(entityTag, S3Constants.XATTR_STR_CHARSET))
-                .setXattrUpdateStrategy(File.XAttrUpdateStrategy.UNION_REPLACE)
-                .build());
+            S3RestUtils.setEntityTag(fs, objectUri, entityTag);
```

Comment:
`S3RestUtils.setEntityTag()` is the helper function for the prior code. This section was missed previously.